### ISSUE TITLE
Add config to set external CDN URL

### ISSUE THREAD

**Issue by CaiJimmy** [CaiJimmy/hugo-theme-stack#78]
Currently, this theme uses JSDelivr for external libraries like PhotoSwipe or Vibrancy.js. All those URLs are hardcoded in HTML file or JS file in case of PhotoSwipe:

https://github.com/CaiJimmy/hugo-theme-stack/blob/54571f036b4abca8d20497c957e493ad1df12516/assets/ts/gallery.ts#L234-L246

I think it's good to add a new field in config.yaml, something like:

```yaml
params:
    external:
        photoswipe: 
            - JS File 1
            - JS File 2
            - CSS 1
            - CSS 2
        vibrancy: JS File 3
```

Because that Hugo does not have config file overwrite feature (?), those URLs will still be hardcoded, but by adding this new section of the configuration, users will be able to change them without modifying the source code.

----------

For the case of PhotoSwipe, the library is loaded asynchronously using JavaScript. To pass the URL, I'm considering using this new feature introduced in **Hugo v0.78.0:**

![image](https://user-images.githubusercontent.com/5889006/102826615-53f1f300-43e1-11eb-88d8-bf8bea86a9e4.png)

https://gohugo.io/news/0.78.0-relnotes/

**Comment by CaiJimmy** [comment]
I'll start work on this once this issue is solved: #70 , which requires me to rewrite `gallery.ts`.

**Comment by CaiJimmy** [comment]
Update: PhotoSwipe libraries are now loaded from `layouts/partials/article/components/photoswipe.html`: https://github.com/CaiJimmy/hugo-theme-stack/blob/master/layouts/partials/article/components/photoswipe.html
